### ISSUE TITLE
Bluesky post with deleted quote post breaks the embed

### ISSUE THREAD

**Issue by umbra3275** [FixTweet/FxTwitter#1100]
Posting an fxbsky link to a post which quote-posts a post that has since been deleted breaks the bot and results in saying "Owie, you crashed FxBluesky (beta) :( ..."

**Comment by dangeredwolf** [comment]
Hi, does this issue still happen?

**Comment by umbra3275** [comment]
yes it does. i'm trying to find some links to share for debugging that are
work-appropriate, but it's a rare situation

**Comment by umbra3275** [comment]
ah, here's one
https://fxbsky.app/profile/mrjimmydafloof.bsky.social/post/3lh3bqqoqxk22

**Comment by umbra3275** [comment]
I'm assuming this is related. When embedding a bluesky post where the poster has quote-posted someone, but also has later blocked the original poster, no images get embedded (despite the fx'd link having an embedded image), and the quote-post is displayed as "Quoting (@)\n\nundefined". Here's an example post, though warning, it is definitely NSFW ||softcore, bulge, furry|| <https://bsky.app/profile/aetherotter.bsky.social/post/3k5kcpa4mul2f>
